src/components/UserSelector.tsx

```diff
@@ -1,6 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import { User } from '../types/User';
+import { getUsers } from '../api/ApiMethods';
+
+type Props = {
+  setSelectUser: (v: User) => void,
+  selectUser: User | null,
+};
+
+export const UserSelector: React.FC<Props> = ({
+  setSelectUser,
+  selectUser,
+}) => {
+  const [allUsers, setAllUsers] = useState<null | User[]>(null);
+  const [dropdownTrigger, setDropdownTrigger] = useState(true);
+
+  const handleTrigger = () => {
+    setDropdownTrigger(prev => !prev);
+  };
+
+  const handleSelect = (user: User) => {
```

Comment:
It should never happen:
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/86923089/b505c193-ee2e-4a6a-a7a5-30d8c6f5965a)
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/86923089/d3261764-067f-4806-80fb-309818d2d053)

Carefully read the requirements and try to pass all tests